config_manager.go

```diff
@@ -35,6 +37,16 @@ type ConfigFile struct {
 	ExcludePatterns *RegexCollection `yaml:"exclude_patterns"`
 }
 
+func (cf ConfigFile) printValidationWarnings() {
+	if len(cf.Files) == 0 {
```

Comment:
This only checks for files passed in via the config. We'll likely want to check the output from [Files()](https://github.com/featalion/remote_syslog2/blob/fix-issue-43/config_manager.go#L305) instead so we know if r_s2 isn't monitoring anything.